frontend/model/contracts/group.js

```diff
@@ -208,6 +297,9 @@ sbp('chelonia/defineContract', {
           recentDate = recentDate.toISOString()
         }
         const { distributionDate, distributionPeriodLength } = getters.groupSettings
+
+        if (!distributionDate) return null
```

Comment:
Why was this added?

As far as I know, this should never happen since the group contract is called with this initiated...

Reply:
@taoeffect This is actually a fix for below issue.

<img src='https://user-images.githubusercontent.com/17641213/204125890-a73dee17-0414-4e11-90bf-9e0879f47d1e.png' width='440'>

Reply:
Oh, ok, thanks, that looks like an OK fix then 👍